packages/core/src/components/IconButton/IconButton.module.scss

```diff
@@ -6,4 +6,16 @@
 
 .referenceWrapper {
   display: inline-flex;
-}
\ No newline at end of file
+}
+
+.loader.loader {
```

Comment:
Why are we providing only a partial set of options? Let's give the ability to show the loader for all sizes, is it by design?

Reply:
It's showing for all of the sizes
We just need an override for the xxs size - 10 instead of 16

Reply:
![image](https://github.com/mondaycom/monday-ui-react-core/assets/104433616/fd39608f-917e-4c0c-a125-6fa7f3c96695)